src/main/java/programminglife/model/drawing/SubGraph.java

```diff
@@ -486,7 +482,7 @@ private void topoSortFromNode(ArrayList<DrawableNode> result,
      * Calculate genomes through edge, based on topological ordering and node-genome information.
      * @return a {@link Map} of {@link Map Maps} of collections of genomes through links
      */
-    Map<DrawableNode, Map<DrawableNode, Collection<Integer>>> calculateGenomes() {
+    public Map<DrawableNode, Map<DrawableNode, Collection<Integer>>> calculateGenomes() {
         Map<DrawableNode, Map<DrawableNode, Collection<Integer>>> genomes = new LinkedHashMap<>();
```

Comment:
Yes, but this 1) makes testing harder and 2) can be done when the new drawing is present (save the genomes in `DrawableEdge` directly.